1999Home.html

```diff
@@ -9,7 +9,7 @@
   <script src=js/home.js></script>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="application-name" content="WebTV HD">
+  <meta name="application-name" content="WebTV HD Fork">
```

Comment:
You're changing every instance of WebTV HD as if the fork is a completely different project. It's not.